api-gateway/src/main/java/com/petclinic/bffapigateway/dtos/VisitDetails.java

```diff
@@ -19,4 +19,8 @@
     private String date = null;
 
     private String description = null;
+
+    private Integer practitionerId = null;
+
+    private Boolean status = null;
```

Comment:
Changing the DTO could cause problems when trying to do requests. Maybe just leave it as is ?